src/reducers/profile-view.js

```diff
@@ -171,6 +171,7 @@ function viewOptionsPerThread(
           return newFunc === undefined ? oldFunc : newFunc;
         };
         return {
+          ...threadViewOptions,
```

Comment:
I made this change because in an upcoming PR I add more things in this object and this makes it easier.